docs/source/_templates/our-class.rst

```diff
@@ -0,0 +1,32 @@
+{{ fullname | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
+   :show-inheritance:
+
+   {% block methods %}
+   {% if methods %}
+   .. rubric:: {{ _('Methods') }}
+
+   .. autosummary::
+      :toctree:
+   {% for item in methods %}
+     {% if item != '__init__' %}    {# __init__ is always documented directly under class name #}
```

Comment:
This works for the `autosummary` table but the `Constructor method.` section is now blank:

![image](https://user-images.githubusercontent.com/54918401/104499088-8fc84480-55aa-11eb-8db1-bada9a05c791.png)

It's a bit strange, because currently it seems the default `python3.9/site-packages/sphinx/ext/autosummary/templates/autosummary/class.rst` template uses the line `.. automethod:: __init__` to explicitly document `__init__` but this is causing `Constructor method.` text to duplicate. 

![image](https://user-images.githubusercontent.com/54918401/104499199-bab29880-55aa-11eb-88d8-72520f5b2480.png)

I tried adding that line to your template and the same duplicating behavior occurs. Is there a method template that Autodoc uses by default? I don't see one in the folder where the default class template is located.

Reply:
I removed `autoclass_content = "both"` from `conf.py` and `__init__` docstring should now be ignored

Reply:
oh ok got it